rados/conn.go

```diff
@@ -117,14 +117,7 @@ func (c *Conn) ListPools() (names []string, err error) {
 			continue
 		}
 
-		tmp := bytes.SplitAfter(buf[:ret-1], []byte{0})
```

Comment:
Shouldn't make any difference because delimeter is {0}, but just pointing out that SplitAfter() is used here and not Split()

Reply:
hmm, yeah. Maybe the trailing `"\x00"` got stripped when converting the C-string to a Go-string?

Reply:
It was, but I'm fairly certain that the code was trying to accomplish the same thing. In one case a buffer w/o a trailing null is passed to GoString and in this case the null is preserved. Normally, a C string will have the null byte but in the cases that used strip the code still did the right thing.
The new code doesn't use GoString anyway so the behavior of GoString is less relevant in this case now, IMO.